scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -379,14 +379,30 @@ class FossIdTest : WordSpec({
                 .expectCreateScan(projectCode, scanCode, vcsInfo)
                 .expectDownload(scanCode)
                 .mockFiles(scanCode)
-            coEvery { service.runScan(USER, API_KEY, scanCode) } returns EntityResponseBody(status = 1)
+            coEvery {
+                service.runScan(
+                    USER, API_KEY,
```

Comment:
I'm surprised that *detekt* doesn't complain here to put these arguments also into separate lines, like below. Would you mind still changing that?